tests/libpod/BUILD.bazel

```diff
@@ -6,7 +6,6 @@ go_library(
         "blackhole.go",
         "certs.go",
         "container.go",
-        "pod.go",
```

Comment:
this remove should be on the first commit as I commented there

Reply:
Agree, updated